test/python/test_transport.py

```diff
@@ -146,7 +146,20 @@ def test_add_species_multi(self):
         self.assertNear(gas1.thermal_conductivity, gas2.thermal_conductivity)
         self.assertArrayNear(gas1.multi_diff_coeffs, gas2.multi_diff_coeffs)
 
-    def test_species_visosities(self):
+    def test_high_pressure_chung_viscosity(self):
+        state = 520, 6e7, 'NH3:1.0'
+
+        gas1 = ct.Solution('gri30.yaml')
```

Comment:
Since you aren't making use of the input file that's automatically loaded for the `TestTransport` class, you might consider adding a new test class, analogous to `TestIonTransport`.